node/pkg/logscribeconsumer/types.go

```diff
@@ -42,3 +48,9 @@ func WithStoreInterval(interval time.Duration) AppOption {
 		c.StoreInterval = interval
 	}
 }
+
+func WithLogscribeEndpoint(endpoint string) AppOption {
+	return func(c *AppConfig) {
+		c.LogscribeEnpoint = endpoint
+	}
+}
```

Comment:
can you also add option `WithStoreLevel()` so that log store level can be configured on initializing?